docs/data/registry.json

```
@@ -96,31 +92,73 @@
       ]
     }
   },
+  "elixir": {
+    "description": "",
+    "repo_name": "elixir",
+    "disk": false,
+    "docs": {
+      "relationship_name": null,
+      "service_name": null,
+      "url": "/languages/elixir.html",
+      "web": {
+        "commands": {
+          "start": "mix run--no-halt"
```

```suggestion
          "start": "mix run --no-halt"
```

Missing space.  Also, this is different than the command in the template.

---

Fixed. Ya this is the general start command, whereas the other uses Ori's config reader + phoenix command